app/policies/RentPolicy/rent_policy.rb

```diff
@@ -0,0 +1,28 @@
+class RentPolicy < ApplicationPolicy
+
+  class Scope < Scope
+    def resolve
+      if user.admin?
```

Comment:
Does the admin? method exists on the User model ?
You may add the following:

```ruby
def admin?
  false
end
```

into your class.

Reply:
done